lib/renaissance_web/controllers/bid_controller.ex

```diff
@@ -0,0 +1,29 @@
+defmodule RenaissanceWeb.BidController do
+  use RenaissanceWeb, :controller
+
+  alias Renaissance.{Bids}
+  alias RenaissanceWeb.Helpers.Auth
+
+  def create(conn, params) do
+    bidder = Auth.current_user(conn)
+    auction_id = String.to_integer(params["auction_id"])
+
+    response =
+      params
+      |> Map.put("auction_id", auction_id)
```

Comment:
Wouldn't a map merge be a bit cleaner?

Reply:
and this is just adding in the int version of auction id? Seems odd?

Reply:
In PR #11 (`bid-frontend` ← `fix-bid-concurrency`), this is changed to 
<img width="543" alt="Screen Shot 2019-05-09 at 3 58 57 PM" src="https://user-images.githubusercontent.com/5504473/57486289-6dc5dd00-7273-11e9-8914-e90d539bdf68.png">
  \[[ link ](https://github.com/xray/Renaissance/blob/fix-bid-concurrency/lib/renaissance_web/controllers/bid_controller.ex#L7) \]

Reply:
Resolved with https://github.com/xray/Renaissance/commit/bd0d702157b55f30fd237005ba264712b535732e#diff-35e23b1cd34500f693ff9083435c3406R25